src/styles/_variables.scss

```diff
@@ -1,26 +1,177 @@
 /* fonts */
 $main-font: 'IBM Plex Sans', sans-serif;
 $second-font: 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$mono-font: Consolas, 'Andale Mono WT', 'Andale Mono', 'Lucida Console',
-  'Lucida Sans Typewriter', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono',
-  'Liberation Mono', 'Nimbus Mono L', Monaco, 'Courier New', Courier, monospace;
 
 $base-font-weight: 400;
 $heavy-font-weight: 700;
 
 /* colors */
-$primary-color: magenta;
+$primary-color: #ff00ff;
 $secondary-color: #006cff;
-$secondary-faded: #bcd8ff;
 
 $dt-bg-team: #1d1f22;
 
-$border-color: rgba(0, 0, 0, 0.08);
+$border-color: #00000014;
 
-$black: black;
-$white: white;
+$black: #000000;
+$white: #ffffff;
 $grey: #f0f0f0;
+$lighten-black: #808080;
+$darken-grey: #343434;
+$lighten-grey: #858585;
 
 /* miscellaneous */
 $assetsPath: '../assets';
 $border-radius: 16px;
+
+:root {
+  /* fonts */
+  --main-font: #{$main-font};
+  --second-font: #{$second-font};
+
+  /* font size */
+
+  --xs-font: 10px;
+  --sm-font: 12px;
+  --md-font: 14px;
+  --lg-font: 16px;
+  --xl-font: 22px;
+  --xxl-font: 26px;
+  --xxxl-font: 32px;
+
+  /* fonts weight */
+  --font-weight-regular: #{$base-font-weight}; //400
+  --font-weight-semi-bold: 600;
+  --font-weight-bold: #{$heavy-font-weight}; //700
+
+  /* colors */
+  --primary-color: #{$secondary-color};
```

Comment:
primary color is set to secondary-color?

Reply:
actually, disregard the above note, and just add a comment to the file here to say why it's being set this way!